docs/parameters.yaml

```diff
@@ -1718,7 +1718,7 @@ description: |+
 
   ```yaml
   Cache:
-    ClientStatisticsLocation: "/tmp/xrootd.stats"
+    ClientStatisticsLocation: "$Cache.RunLocation/xrootd.stats"
```

Comment:
We're not 100% consistent in our parameters yaml, but I think our convention is to use the `${}` bash-ism for describing multi-level configs elsewhere in our docs. Can you switch this to `${Cache.RunLocation}`?